client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -954,13 +946,17 @@ export default {
 
     triggerStatementDeletion (id) {
       if (window.confirm(Translator.trans('check.statement.delete'))) {
+        // Override the default success callback to display a custom message
+        this.$store.api.successCallbacks[0] = async (success) => this.$store.api.handleResponse(success, {
+          200: { type: 'confirm', text: Translator.trans('confirm.statement.deleted') },
+          204: { type: 'confirm', text: Translator.trans('confirm.statement.deleted') }
+        })
+
```

Comment:
Is this how we need to configure custom messages for the vuex json api lib requests?

Reply:
exactly.
At least I didn't find a leaner way